src/resolve.rs

```diff
@@ -162,19 +232,22 @@ impl ResolveValue for ast::Expression {
                     }
                 }
 
-                select_default(variants)?.value.to_value(env)
+                select_default(variants)
+                    .ok_or(FluentError::None)?
+                    .value
+                    .to_value(env)
             }
             ast::Expression::CallExpression {
                 ref callee,
                 ref args,
             } => {
-                let mut resolved_unnamed_args = Vec::new();
-                let mut resolved_named_args = HashMap::new();
+                let resolved_unnamed_args = &mut Vec::new();
+                let resolved_named_args = &mut HashMap::new();
```

Comment:
Is that now a preferred convention?

Reply:
It's less a convention and more of a requirement in some situations, when you want to ensure that the borrow checker understands that you only want to pass a reference to a closure, rather than having the closure take ownership of the value you want to reference within.

It can be more idiomatic in some situations though, when you know that you're going to just borrow the thing repeatedly.